server/github_pr.go

```diff
@@ -20,13 +21,45 @@ type PullRequestEvent struct {
 	RepositoryURL string              `json:"repository_url"`
 }
 
-type IssueComment struct {
+type EventData struct {
 	Action     string                     `json:"action"`
 	Comment    *github.PullRequestComment `json:"comment"`
 	Issue      *github.Issue              `json:"issue"`
 	Repository *github.Repository         `json:"repository"`
 }
 
+// CheckCLA is true if body contains "/check-cla"
+func (d *EventData) CheckCLA() bool {
+	if d.Comment == nil || d.Comment.Body == nil {
+		return false
+	}
+	return strings.Contains(strings.TrimSpace(*d.Comment.Body), "/check-cla")
```

Comment:
What do you think of using regexp to be sure using something like `\b(/check-cla)\b`? 2/5

Reply:
Don't have any strong opinions, I tend to use regexp on more complex stuff :)

Reply:
A simple `strings.Contains` is fine IMO.